Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.9
+FROM alpine:3.10
 
 ARG KUBE_VERSION="v1.10.11"
```

Comment:
ohhh this should be 1.12.10

Reply:
Updated! Is it safe to merge this, are all of our clusters etc compatible w/ the `1.12.10` version yet?

Reply:
> Updated! Is it safe to merge this, are all of our clusters etc compatible w/ the `1.12.10` version yet?

sounds like we should be fine:https://kubernetes.io/docs/setup/release/version-skew-policy/#kubectl

clusters are either 1.11 or 1.12